ivadomed/loader/utils.py

```diff
@@ -29,6 +29,13 @@
 TRANSFORM_PARAMS = ['elastic', 'rotation', 'scale', 'offset', 'crop_params', 'reverse',
                     'translation', 'gaussian_noise']
 
+# Ordered list of supported file extensions
+# TODO: Implement support of the following OMETIFF formats (#739):
+# [".ome.tif", ".ome.tiff", ".ome.tf2", ".ome.tf8", ".ome.btf"]
+# They are included in the list to avoid a ".ome.tif" or ".ome.tiff" following the ".tif" or ".tiff" pipeline
+EXT_LST = [".nii", ".nii.gz", ".ome.tif", ".ome.tiff", ".ome.tf2", ".ome.tf8", ".ome.btf", ".tif",
+           ".tiff", ".png", ".jpg", ".jpeg"]
```

Comment:
I added the list to the error message in 7e26faf.
I agree that we should also update the documentation but I think it would be best to do it in "data" [here](https://ivadomed.org/en/latest/data.html). I would prefer to wait until the microscopy pipeline is complete before doing that. I also plan on adding a tutorial for microscopy.